src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpuploadresponse/DwpUploadResponseAboutToSubmitHandlerTest.java

```diff
@@ -64,6 +65,7 @@ public void givenADwpUploadResponseEvent_thenSetCaseCode() {
         PreSubmitCallbackResponse<SscsCaseData> response = dwpUploadResponseAboutToSubmitHandler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
 
         assertEquals("002DD", response.getData().getCaseCode());
+        assertEquals(LocalDate.now().toString(), response.getData().getDwpResponseDate());
```

Comment:
It is a date not a string

Reply:
This is the pattern we store all dates in CCD as